internal/driver/glfw/window_darwin.go

```diff
@@ -0,0 +1,23 @@
+//go:build darwin
+
+package glfw
+
+import (
+	"fyne.io/fyne/v2/driver"
+)
+
+// assert we are implementing driver.NativeWindow
+var _ driver.NativeWindow = (*window)(nil)
+
+func (w *window) RunNative(f func(any) error) error {
+	var err error
+	done := make(chan struct{})
+	runOnMain(func() {
+		err = f(driver.MacWindowContext{
+			NSWindow: uintptr(w.view().GetCocoaWindow()),
```

Comment:
The call to `w.view()` can return nil. This might crash if called at the wrong time? Applies to most of the implementations.

Reply:
Fixed - they will now pass a null uintptr to the callback if executed before the window is created